public/views/app.html

```diff
@@ -97,18 +113,44 @@ <h5 >
     
       </div> 
     </section>
-   <aside class="players">
-    <!-- Search form for email-->
-      <form ng-show="game.state === 'awaiting players' && showOptions === false" class="search-form search-form-side" name="searchForm">
-        <input  placeholder="Search for players..." ng-model="user.email"/>
-        <button type="submit" ng-click="getUserInfo(user, searchForm)" >
-            <i class="material-icons">search</i>
-        </button>
-      </form> 
-      <div player style="width:100%">
-
+    <aside class="players">
+      <div class="row" ng-controller="IndexController">
+        <div class="col s12">
+          <ul class="tabs">
+            <li class="tab">
+              <a class="active" href="#" ng-click="showtab1()">Players</a>
```

Comment:
I believe these (`showtab1()`, `showtab2()`) should also be corrected based on feedback you had received.

Reply:
I have implemented it